sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample8_ModelCompose.md

```diff
@@ -0,0 +1,128 @@
+# Model compose
+
+Model compose allows multiple models to be composed and called with a single model ID. This is useful when you have trained different models and want to aggregate a group of them into a single model that you (or a user) could use to recognize a form.
+When doing so, you can let the service decide which model more accurately represents the form to recognize, instead of manually trying each trained model against the form and selecting the most accurate one.
+
+This sample demonstrates how to:
+- Create a composed model from existing models.
+- Differentiate between a composed model and a single model.
+- Recognize a custom form using a composed model.
+- Explore the models that are part of a composed model.
+
+Please note that composed models can also be created using a graphical user interface such as the [Form Recognizer Labeling Tool][labeling_tool].
+
+To get started you'll need a Cognitive Services resource or a Form Recognizer resource.  See [README][README] for prerequisites and instructions.
+
+## Creating a `FormTrainingClient`
+
+Model compose management is located in the `FormTrainingClient`. To create a new `FormTrainingClient` you need the endpoint and credentials from your resource. In the sample below you'll use a Form Recognizer API key credential by creating an `AzureKeyCredential` object, that if needed, will allow you to update the API key without creating a new client.
+
+You can set `endpoint` and `apiKey` based on an environment variable, a configuration setting, or any way that works for your application.
+
+```C# Snippet:CreateFormTrainingClient
+string endpoint = "<endpoint>";
+string apiKey = "<apiKey>";
+var credential = new AzureKeyCredential(apiKey);
+var client = new FormTrainingClient(new Uri(endpoint), credential);
+```
+
+## Create a composed model
+In our case, we will be writing an application that collects the expenses a company is making. There are 4 main areas where we get purchase orders from (office supplies, office equipment, furniture, and cleaning supplies). Because each area has its own form with its own structure, we need to train a model per form.
+
+```C# Snippet:FormRecognizerSampleTrainVariousModels
+// For this sample, you can use the training forms found in the `trainingFiles` folder.
```

Comment:
Did you actually include training files for all these separate forms?

Reply:
I used the same training files for every purchase model, so the information under `trainingFiles` still applies